src/include/function/export/export_function.h

```diff
@@ -54,8 +58,9 @@ using bind_t =
     std::function<std::unique_ptr<ExportFuncBindData>(function::ExportFuncBindInput& bindInput)>;
 using init_local_t = std::function<std::unique_ptr<ExportFuncLocalState>(main::ClientContext&,
     const ExportFuncBindData&, std::vector<bool>)>;
-using init_shared_t = std::function<std::shared_ptr<ExportFuncSharedState>(main::ClientContext&,
-    ExportFuncBindData&)>;
+using create_shared_t = std::function<std::shared_ptr<ExportFuncSharedState>()>;
```

Comment:
These type defines are globally visible. Just like we did for scalar_func_exec_t, better rename as export_func_xxx_t